THIRD_PARTY_NOTICES_ADDENDUM.md

```diff
@@ -0,0 +1,204 @@
+## Third Party License Addendum
```

Comment:
You have both a third party licenses and a third party notices addendum. I don't think you need both of these.

Reply:
👍 just deleted the other THIRD_PARTY_LICENSES_ADDENDUM.md file